src/component/mxgraph/StyleConfigurator.ts

```diff
@@ -144,9 +146,14 @@ export default class StyleConfigurator {
     this.putCellStyle(ShapeBpmnElementKind.GATEWAY_PARALLEL, style);
   }
 
-  private configureExclusiveGatewayStyle(): void {
-    const style = this.mxUtils.clone(this.getStylesheet().getCellStyle(ShapeBpmnElementKind.GATEWAY_PARALLEL), this.getDefaultVertexStyle());
-    style[this.mxConstants.STYLE_GRADIENTCOLOR] = '#DDA0DD';
-    this.putCellStyle(ShapeBpmnElementKind.GATEWAY_EXCLUSIVE, style);
+  private configureGatewaysStyle(): void {
+    ShapeUtil.topLevelBpmnGatewayKinds().forEach(kind => {
+      const style = this.cloneDefaultVertexStyle();
+      style[this.mxConstants.STYLE_SHAPE] = kind;
+      style[this.mxConstants.STYLE_PERIMETER] = this.mxPerimeter.RhombusPerimeter;
```

Comment:
Maybe no need that, as GatewayShape extends mxRhombus. To test

Reply:
Probably same as for event, see https://github.com/process-analytics/bpmn-visualization-js/pull/184#pullrequestreview-397352288

Reply:
could not find that is set up automatically
I found that in their examples (monitor, swimlanes) they set it up explicitly
and in fact, we set up the properties of style here (not the shape)

Reply:
As for event, I suggest we keep it